modelling/src/main/java/org/axonframework/modelling/saga/AbstractResourceInjector.java

```diff
@@ -42,6 +42,7 @@ public abstract class AbstractResourceInjector implements ResourceInjector {
 
     private static final String[] DEFAULT_INJECT_ANNOTATIONS = {
             "javax.inject.Inject",
+            "jakarta.inject.Inject",
```

Comment:
Having a test for this would be nice.

Reply:
That could be troublesome, I guess. You'd need to add `jakarta` packages to this non-jakarta module, for no other purpose but this test.  

Also, is it really needed? I assume there is a test for the `javax` package. If so, it is automatically copied to the respective `jakarta` module, transformed, and run there. So effectively, there is a test for `jakarta.inject.Inject` as well. Just that we don't have the source code for it.

Reply:
You're right, the `SimpleResourceInjectorTest` should pick this up. Good thinking :+1: